ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/deneb/helpers/MiscHelpersDeneb.java

```diff
@@ -236,4 +248,15 @@ public int getBlobKzgCommitmentsCount(final SignedBeaconBlock signedBeaconBlock)
         .map(SszList::size)
         .orElse(0);
   }
+
+  public boolean verifyBlobSidecar(final BlobSidecar blobSidecar) {
+    try {
+      return kzg.verifyBlobKzgProofBatch(
+          List.of(blobSidecar.getBlob().getBytes()),
+          List.of(blobSidecar.getKZGCommitment()),
+          List.of(blobSidecar.getKZGProof()));
+    } catch (final KZGException ex) {
+      return false;
```

Comment:
Another thing is that I'm not a fan of swallowing the exception here without even a log line (debug?)